README.md

```diff
@@ -1,3 +1,28 @@
-# tpm_module
+# TPM2 module schematics
 
-TPM module PCB project for Libretrend LT1000
\ No newline at end of file
+TPM2 module PCB project for platforms with 2x5 pin, 2mm pitch LPC header, e.g.
+Libretrend LT1000.
```

Comment:
@artur-rs I would definitely write that this TPM was created for Librebox and under contract work from Libretrend. Let's avoid any confusion that we did it on our own without being compensated.

Also, the question is if pin layout is important in this case? Or it is always the same? If it is uniq we should probably elaborate that and point to graphics. BTW I think image of pin layout should be here and attached to social media announcement.

Reply:
@pietrushnic please check if now is ok